docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -83,15 +83,15 @@ When accessing the executor Pod's logs the following error is seen.
 
 *What it means:*
 
-A `NoRouteToHostException` indicates that this Pod is not able to connect to the {WebName} Pod.
+A `NoRouteToHostException` indicates that this pod is not able to connect to the `{WebName}` pod.
```

Comment:
I think you should remove the "What it means" title and explanation because (1) it's obvious from the error message and (2) it's not entirely accurate. The error states that there is no route to the host. While the consequence of this state is that the executor cannot connect to the web console, strictly speaking, that is not what the error means. \

I would only include an explanation of the error message if the error message is badly written or unintelligible. And then I would submit a bug ASAP to the developer to fix the log error. Same for misspelled words, bad grammar, anything that looks bad. 

Remember that tech writing is another level of QE. QE checks functionality but we're responsible for anything language-related, including logs, tooltips, and UI labels.